package.json

```diff
@@ -30,7 +30,12 @@
     "install:server": "cd server && yarn install"
   },
   "dependencies": {
+    "@discordjs/rest": "^2.0.1",
```

Comment:
Why are these being added to the root package.json? These should only exist inside the discord_bot/package.json file.

Reply:
Should I remove that then?